src/cpp/src/EXAMPLE/check_missing_cols.cc

```diff
@@ -14,35 +14,45 @@ static std::vector<std::string> FILEN ;
 int main(int argc, char** argv ){
 
   // read file names from command line (only argument) 
+  bool minimal = false ;
   if( argc < 2) {
     std::cout << " usage:  check_missing_cols <input-file1> [[input-file2],...]" << std::endl << std::endl ;
```

Comment:
```suggestion
    std::cout << " usage:  check_missing_cols [--minimal] <input-file1> [[input-file2],...]" << std::endl << std::endl ;
```
Also update the usage string to reflect the new capabilities.